### PR TITLE
OSDOCS#6500: AWS - move install config parameters out of assemblies

### PR DESCRIPTION
**Pull request by mjpytlak**
Version(s):
4.14+

Issue:
This PR addresses [osdocs-6500](https://issues.redhat.com/browse/OSDOCS-6500).

Link to docs preview:
- [Installation configuration parameters for AWS](https://66099--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_aws/installation-config-parameters-aws#installation-configuration-parameters_installation-config-parameters-aws)
  New assembly at book level.
- Installing a cluster on AWS with customizations > [Creating the installation configuration file](https://66099--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_aws/installing-aws-customizations#installation-initializing_installing-aws-customizations)
Note the absence of Installation configuration parameters following this topic. In lieu of the module, a cross reference is provided for the new assembly.

Given that AWS is the last provider in this refactoring effort, and an adjustment was made to the headings in `modules/installation-configuration-parameters` [here](https://github.com/openshift/openshift-docs/pull/66099/files#diff-14ebe576efb84b2f58b32a0de2996fced237204ae9b091e3cafbd40b65c1ef31L96), it is worth noting, that I also verified that the refactored content continues to publish correct for the providers that were previously refactored:

- [Alibaba](https://66099--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_alibaba/installation-config-parameters-alibaba)
- [Azure](https://66099--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_azure/installation-config-parameters-azure)
- [Azure Stack Hub](https://66099--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_azure_stack_hub/installation-config-parameters-ash)
- [GCP](https://66099--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_gcp/installation-config-parameters-gcp)
- [IBM Cloud VPC](https://66099--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_ibm_cloud_public/installation-config-parameters-ibm-cloud-vpc)
- [Nutanix](https://66099--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_nutanix/installation-config-parameters-nutanix)
- [Bare metal](https://66099--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_bare_metal/installation-config-parameters-bare-metal)
- [IBM Z and IBM LinuxOne](https://66099--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_ibm_z/installation-config-parameters-ibm-z)
- [IBM Power](https://66099--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_ibm_power/installation-config-parameters-ibm-power)
- [IBM Power Virtual Server](https://66099--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_ibm_powervs/installation-config-parameters-ibm-power-vs)
- [OpenStack](https://66099--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_openstack/installation-config-parameters-openstack)
- [vSphere](https://66099--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_vsphere/installation-config-parameters-vsphere)

QE review:
- [N/A] QE has approved this change.
Given that this update is a reorg only, QE approval is not required. See conversation at https://github.com/openshift/openshift-docs/pull/61165#pullrequestreview-1493117699

### CODE REVIEW

File: installing/installing_aws/installation-config-parameters-aws.adoc [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+= Installation configuration parameters for AWS
+include::_attributes/common-attributes.adoc[]
+:context: installation-config-parameters-aws
+:platform: AWS
+
+toc::[]
+
+Before you deploy an {product-title} cluster on AWS, you provide parameters to customize your cluster and the platform that hosts it. When you create the `install-config.yaml` file, you provide values for the required parameters through the command line. You can then modify the `install-config.yaml` file to customize your cluster further.
+
+include::modules/installation-configuration-parameters.adoc[leveloffset=+1]
```
Comment (mjpytlak) [inline installing/installing_aws/installation-config-parameters-aws.adoc:12]: **Peer review**

The goal of this PR is to refactor existing content. I acknowledge that this existing content needs some work. Once I have completed the refactor for all providers, I plan on rewriting this abstract and pushing it through QE review. Please see previous reviews:

- https://github.com/openshift/openshift-docs/pull/64049#discussion_r1310262310
- https://github.com/openshift/openshift-docs/pull/64103#discussion_r1311755915
- https://github.com/openshift/openshift-docs/pull/61722#discussion_r1247021585

File: modules/installation-configuration-parameters.adoc [hunk 2]
```diff
@@ -86,53 +50,19 @@ endif::[]
 ifeval::["{context}" == "installation-config-parameters-azure"]
 :azure:
 endif::[]
-ifeval::["{context}" == "installing-restricted-networks-azure-installer-provisioned"]
-:azure:
```
Comment (mjpytlak) [inline modules/installation-configuration-parameters.adoc:90]: Peer review

This `ifeval` is no longer required and is left over from https://github.com/openshift/openshift-docs/pull/61086. The Azure restricted installation assembly correctly references Installation configuration parameters for Azure.

**Comment by sjstout** [comment]
lgtm!

**Comment by bscott-rh** [comment]
LGTM, thanks Mike.

**Pull request merged**